src/pages/Dashboard/Modals/BorrowRepay/Repay/index.tsx

```diff
@@ -258,6 +267,7 @@ const Repay: React.FC<IRepayProps> = ({ asset, onClose, isXvsEnabled }) => {
             isXvsEnabled={isXvsEnabled}
             isRepayLoading={isRepayLoading}
             limitTokens={limitTokens.toFixed()}
+            dailyXvsDistributionInterestsCents={dailyXvsDistributionInterestsCents}
```

Comment:
We're drilling this prop a lot (and some others too, but that's an issue for another day). We can avoid doing that by creating a hook that calls `useGetVTokenDailyXvsWei` and returns `dailyXvsDistributionInterestsCents` so that it can be imported and called everywhere we need it (Borrow/Repay form, Supply/Withdraw form, My Account section).